test/test_gosu_module.rb

```diff
@@ -0,0 +1,112 @@
+# Encoding: UTF-8
+
+require "minitest/autorun"
+require "gosu" unless defined? Gosu
+
+class TestGosuModule < Minitest::Test
+  def test_drawing_primitives
+    {
+      # args: [x, y, width, height, c, z, mode]
+      draw_rect: [0, 0, 10, 10, 0xff_ffffff, 0, :default],
+
+      # args: [x, y, c]{2,4}, z, mode
+      draw_line: [0, 0, 0xff_ffffff, 10, 10, 0xff_ffffff, 0, :default],
+      draw_triangle: [0, 0, 0xff_ffffff, 10, 10, 0xff_ffffff, 0, 10, 0xff_ffffff, 0, :default],
+      draw_quad: [0, 0, 0xff_ffffff, 10, 10, 0xff_ffffff, 0, 10, 0xff_ffffff, 0, 10, 0xff_ffffff0, 0, :default],
+    }.each do |method, args|
+      assert_respond_to Gosu, method
+
+      img = Gosu.record(20, 20) do
+        Gosu.send(method, *args)
+      end
+      assert_equal Gosu::Image, img.class
+
+      # RuntimeError: Gosu::Macro cannot be rendered as Gosu::Bitmap yet
+      # that would be awesome. Making "screenshots" in general would greatly
+      # extend the possibilities of automated testing.
+      # Already requestesd in https://github.com/gosu/gosu/issues/317
+      # img.save("/tmp/test.png")
+    end
+  end
+
+
+  # TODO: Manipulating the current drawing context
+  # clip_to  flush  gl  record  rotate  scale  transform  translate
+
+  def test_misc
+    assert_match(/^[a-z]{2}/, Gosu.language)
+
+    assert_equal 0, Gosu.fps
+    assert_equal 0, Gosu.milliseconds
+
+    assert_equal 'sans', Gosu.default_font_name
```

Comment:
This will fail on Windows/macOS where the default font is "Arial".

Do you know enough about Linux' font loading to say whether "Arial" could be used there as well?